### DISCUSSION TITLE
How to programatically control media? E.g. seek the media element to X seconds on a button click.

### DISCUSSION THREAD

**Discussion by rssfrncs** [muxinc/media-chrome#890] [Q&A] [answered]
Hey 👋🏻 

I'm wondering if it's possible to programmatically control the media-chrome elements e.g. seek the media element to X seconds on a button click. I can do this when using the React components and the react/media-store hooks but wondering if it's possible with the vanilla elements?

**Comment by rssfrncs** [comment]
Am i overthinking this and simply I need to view the custom video elements .g. `<hls-video-element />` as the standard html `<video /> element and call getters/setters such as `element.volume` or `element.currentTime = 10`?

**Reply by luwes** [reply, marked as answer]
it's a simple as that 👍 
that's the goal of all the custom media elements, to aim for a HTMLMediaElement compatible API

**Reply by rssfrncs** [reply]
awesome thanks for such a great lib!
